rust/src/domains/polars/expr/mod.rs

```diff
@@ -178,8 +205,17 @@ impl Debug for ExprDomain {
     }
 }
 
+/// OuterMetric encodes the relationship between
```

Comment:
not a proof definition?

Reply:
Yeah, this one isn't worded as a proof definition. There are some cases where a proof definition on the trait itself could make sense (for marker traits), but I think most of the time it's the methods, consts and associated types that need definitions.